engine/src/main/java/org/terasology/engine/rendering/world/RenderableWorldImpl.java

```diff
@@ -46,7 +46,7 @@
     private static final int MAX_ANIMATED_CHUNKS = 64;
     private static final int MAX_BILLBOARD_CHUNKS = 64;
     private static final int MAX_LOADABLE_CHUNKS = ViewDistance.MEGA.getChunkDistance().x() * ViewDistance.MEGA.getChunkDistance().y() * ViewDistance.MEGA.getChunkDistance().z();
-    private static final Vector3f CHUNK_CENTER_OFFSET = new Vector3f(0.5f, 0.5f, 0.5f);
+    private static final Vector3f CHUNK_CENTER_OFFSET = new Vector3f(Chunks.CHUNK_SIZE).div(2);
```

Comment:
this variable is derived fro Chunks so would it make center to move it there? also use Vector3fc to guard the object.

Reply:
I don't feel like having this in `Chunks` would particularly make it better. The implementation is really obvious and I don't expect it to be re-used all that much. Regarding `Vector3fc`: okay, I'll change that.